getting-started/k8s/README.md

```diff
@@ -0,0 +1,93 @@
+# Getting Started with linkerd and Kubernetes
```

Comment:
I like this suggestion.  @olix0r: are there any external links to gob that would break if we moved k8s and dcos into gob?